src/components/Tabs/Tabs.jsx

```diff
@@ -47,10 +60,11 @@ class Tabs extends React.Component {
   }
 
   render() {
+    const tabsStyle = this.getTabsStyle();
```

Comment:
I think no need for variable here since it doesn't add any clarity, can just use inline